dev-client/src/screens/SoilScreen/components/EditIntervalModalContent.tsx

```diff
@@ -132,6 +128,24 @@ export const EditIntervalModalContent = ({
     [schema, dispatch, onClose, siteId],
   );
 
+  const userRole = useSiteRoleContext();
+
+  const editingAllowed = useMemo(() => {
```

Comment:
question: should this be `isEditingAllowed`? or `canEdit`? How do we name similar functions elsewhere?

Reply:
I don't know if we have decided on a convention for this. I am impartial either way.